src/Faker/Provider/cs_CZ/Person.php

```diff
@@ -425,6 +427,66 @@ class Person extends \Faker\Provider\Person
         'Bc.', 'Ing.', 'MUDr.', 'MVDr.', 'Mgr.', 'JUDr.', 'PhDr.', 'RNDr.', 'doc.', 'Dr.'
     );
 
+    /**
+     * @return czech birth number
+     * @param string|null $gender 'male', 'female' or null for any
+     * @param int $minAge minimal age of "generated person" in years
+     * @param int $maxAge maximal age of "generated person" in years
+     */
+
+    public function birthNumber($gender = null, $minAge = 0, $maxAge = 100, $slashProbability = 50)
+    {
+        if ($gender == null) {
+            $gender = Miscellaneous::boolean() ? static::GENDER_MALE : static::GENDER_FEMALE;
```

Comment:
Don't use the static `boolean()` provider - this prevents users from overriding it. Instead, please use `$this->generator->boolean()`.